### ISSUE TITLE
Allow section padding to be defined per break point

### ISSUE THREAD

**Issue by Gibsonion** [soflyy/oxygen-bugs-and-features#1388]
**The Feature**
It currently requires custom code to set section padding that works at different device widths, it would be good to be able to set the global section padding for each break point separately, as one size never fits all sizes.

**What are the use cases for this feature?**
Better responsive designs with a more standardised layout without the need for custom CSS rules.

**Comment by aleklisefski** [comment]
+ 1,000,000

**Comment by PaRoxUs** [comment]
I would appreciate it if all the global styles could be applied based on the breakpoints!

**Comment by simonclay** [comment]
+1

The Comparison page boasts 'Responsive Control for All CSS Properties'. But not if it's Global?

**Comment by brettredvers** [comment]
100%
Is there any reason why responsive settings don't work on global section styles?

I'm surprised more people haven't commented on this.

**Comment by forjoyilive** [comment]
Agreed, this would be very helpful! I decided to stop using sections and their padding partly because of this.
